plugins/inputs/win_wmi/README.md

```diff
@@ -0,0 +1,280 @@
+# Windows Management Instrumentation Input Plugin
+
+This document presents the input plugin to read WMI classes on Windows
+operating systems. With the win_wmi plugin, an administrator is enabled to
+capture and filter virtually any configuration or metric value exposed through
+the Windows Management Instrumentation
+([WMI](https://learn.microsoft.com/en-us/windows/win32/wmisdk/wmi-start-page))
+service.

```

Comment:
Is there a minimum version of Windows required?

Reply:
WMI was introduced in Windows 2000. All operating systems which did not support WMI are long out of service.